backend/app/datasets/routes.py

```diff
@@ -111,10 +111,6 @@ def post(self):
         # Holds a list of tag ids to be associated with the dataset.
         tags = req_body.pop('tags', [])
 
-        # Ensure that new datasets have citations
-        if req_body.get('citation') is None:
-            return {'errors': 'Citation missing'}
-
```

Comment:
Why was this removed? Is it fixed to be handled in the frontend?

Reply:
It should be handled in the frontend.